index.js

```diff
@@ -24,39 +26,46 @@ module.exports = {
 
     if (!(app.options['materialize-shim'] || {}).omitJS) {
       app.import('vendor/materialize/materialize.js');
-      app.import('vendor/materialize-shim.js', {
-        exports: {
-          materialize: ['default']
-        }
+      app.import('vendor/materialize-shim.js');
+    }
+
+    if (!(app.options['materialize-shim'] || {}).omitFonts) {
+      getFontPaths().robotoFiles.forEach(font => {
+        app.import(`vendor/fonts/roboto/${font}`, {
```

Comment:
I added this code because for some reason the fonts weren't getting into the bundle. Try taking the previous version and running `ember build` and looking in `dist` or doing `ember serve` and observing the 404s in the dev tools. There's probably a more elegant way to do this, but this at least seems to work.